tpm2/tpm2.go

```diff
@@ -1187,12 +1187,18 @@ func NVUndefineSpace(rw io.ReadWriter, ownerAuth string, owner, index tpmutil.Ha
 }
 
 // NVUndefineSpaceSpecial This command allows removal of a platform-created NV Index that has TPMA_NV_POLICY_DELETE SET.
-func NVUndefineSpaceSpecial(rw io.ReadWriter, index, platformPP tpmutil.Handle) error {
-	cmd, err := tpmutil.Pack(index, platformPP)
+func NVUndefineSpaceSpecial(rw io.ReadWriter, nvIndex, sessionIndex tpmutil.Handle, pAuth []byte) error {
+	indexAuth := AuthCommand{Session: sessionIndex, Attributes: AttrContinueSession, Auth: []byte(EmptyAuth)}
+	platformAuth := AuthCommand{Session: HandlePasswordSession, Attributes: AttrContinueSession, Auth: []byte(pAuth)}
```

Comment:
Platform hierarchy may set up a special policy that allows certain `POLICY_DELETE` NV indices to be deleted, without revealing the full PlatformAuth (which is likely risky) - if we take this as an `AuthCommand` as well, this will help make that possible.